autogen/src/main.rs

```diff
@@ -76,16 +167,459 @@ async fn update_json() -> Result<()> {
   Ok(())
 }
 
+fn update_gql() -> Result<()> {
```

Comment:
maybe put all the graphql stuff to separate module?

Reply:
moved all the gql code to another file